Builds/levelization/results/loops.txt

```diff
@@ -43,6 +46,9 @@ Loop: ripple.nodestore ripple.overlay
 Loop: ripple.overlay ripple.rpc
   ripple.rpc ~= ripple.overlay
 
+Loop: test.jtx test.rpc
+  test.rpc > test.jtx
+
```

Comment:
Similar to above, this change indicates that a circular dependency was introduced between `src/test/jtx` and `src/test/rpc`. That's not great, though it's less of a problem in test code. It would be a lot better if the relevant places (which I haven't identified yet) were refactored so that nothing in `src/test/jtx` depended on anything in `src/test/rpc`.

See also: https://github.com/XRPLF/rippled/blob/develop/Builds/levelization/README.md